cmd/editor_config.go

```
@@ -0,0 +1,141 @@
+		config.Logger.Error("Version from config file is not the same as the version of the binary")
+		config.Logger.Error("Binary: %s, Config: %s", editorConfigVersion, config.Version)
+		os.Exit(1)
+	}
+
+	if handleReturnableFlags(config) {
+		return
+	}
+
+	filePaths, err := files.GetFiles(config)
+	if err != nil {
+		config.Logger.Error(err.Error())
```

golangci-lint (1.62.2)

printf: non-constant format string in call to (github.com/editorconfig-checker/editorconfig-checker/v3/pkg/logger.Logger).Error

(govet)